tests/ChatTests/Direct.hs

```diff
@@ -1576,12 +1578,19 @@ testUserPrivacy =
       alice <## "alisa (active, hidden, muted)"
       -- hidden message is saved
       alice ##> "/tail"
+      alice <##? chatHistory
+      alice ##> "/_get items count=10"
+      alice <##? chatHistory
+      alice ##> "/_get items before=9 count=10"
```

Comment:
I think it'd be better to test in a separate test where there're both direct and group chats and not in this one, but it's ok